Sources/LiveViewNative/BuiltinRegistry.swift

```diff
@@ -46,6 +46,8 @@ struct BuiltinRegistry {
             Shape(element: element, context: context, shape: RoundedRectangle(from: element))
         case "lvn-link":
             Link(element: element, context: context)
+        case "texteditor":
```

Comment:
If we go with #154 can you rename this to `text-editor`?

```suggestion
        case "text-editor":
```